packages/ui-components/src/components/Icon/Icon.component.tsx

```diff
@@ -892,7 +892,7 @@ export const Icon = forwardRef<HTMLAnchorElement | HTMLButtonElement, IconProps>
   }
 )
 
-Icon.displayName = "IconTs"
+Icon.displayName = "Icon"
```

Comment:
I was wondering if it’s possible to set the display name dynamically instead of hardcoding it. For example:
```js
Icon.displayName = Icon.name
```
or even with a fallback
```js
Icon.displayName = Icon.name || "Icon"
```

Reply:
I think this goes beyond the scope of this PR. Let’s focus on the changeset for now and discuss topics like dynamic or static approaches later. Here’s something to read in the meantime: [MDN: Function.displayName](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/displayName).

Reply:
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Function/displayName#changing_displayname_dynamically

Reply:
The comment is meant to verify if it makes sense and, if so, to create a task for the future.